DToCToDB.md

```diff
@@ -0,0 +1,1193 @@
+
+# Data Cleaning and Export to SQLite3
+###  Using NHS England - Delayed Transfers of Care (DToCs) Datasets
+---
+This notebook describes and demonstrates the steps that I have taken to clean, combine and prepare a dataset ready for export into a SQLite3 Database.
+
+This process has been undertaken using Python with the pandas and SQLite3 libraries. My approach is not Pythonic, however it does work.
+
+The datasets I have used are from the [NHS England](https://www.england.nhs.uk/statistics/) statistical publications. The [National Health Service (England)](https://en.wikipedia.org/wiki/National_Health_Service_(England)), commonly abreviated to the NHS, is the publically funded healthcare system in England (with similar, but seperate systems for other parts of the UK). They collect large amounts of data about the provision of services across different providers (e.g. hospitals). One of these is in regards to [Delayed Transfers of Care (DToCs)](https://www.kingsfund.org.uk/topics/measurement-and-performance/delayed-transfers-care-quick-guide), often referred to in the media as bed blocking. DToCs are when an adult patient is ready for transfer or discharge, yet remain in a hospital bed. There can a range of reasons for this; for example a patient might have been treated for a fractured hip, and could be transfered into a nursing home, but there is no availabilty of beds in the nursing home for them to be transferred into. This is seen as a big challenge in the NHS, therefore the data is collected. Lets look at example of this dataset:
+
+
+```python
+import pandas as pd
+```
+
+
+```python
+oct_16 = pd.read_csv('https://www.england.nhs.uk/statistics/wp-content/uploads/sites/2/2016/06/Monthly-SITREPs-DTOC-Extracts-gf04L.csv', header=3)
```

Comment:
It's perfectly fine to use a remote data source, but if there's a chance the data file could change or go away, it's better to download it locally and bundle it.  This ensures reproducibility of the script.